test/SIL.XForge.Identity.Tests/Controllers/Account/AccountControllerTests.cs

```diff
@@ -145,6 +184,7 @@ public TestEnvironment()
             public IAuthenticationService AuthService { get; }
             public IEventService Events { get; }
             public AccountController Controller { get; }
+            public MemoryRepository<UserEntity> users {get; set;}
```

Comment:
Change this property name to `Users` instead of `users`.

Reply:
Yes, we change users into Users.